_locales/zh/messages.json

```
@@ -0,0 +1,393 @@
+        "placeholders": { "at1": { "content": "<a target='_blank' href='https://dimden.dev/donate/'>" }, "at2": { "content": "</a>" } }
+    },
+    "something_went_wrong": { "message": "出了些问题" },
+    "notifications_error": { "message": "由于错误，某些通知无法加载。" },
+    "tweet_error": { "message": "由于错误，某些推文无法加载。" },
+    "error_instructions": {
+        "message": "请复制下面的文本并将其发送到$AT1$问题跟踪器$AT2$或$AT3$我的电子邮件$AT2$。 谢谢你！",
+        "placeholders": {
+            "at1": { "content": "<a target='_blank' href='https://github.com/dimdenGD/OldTwitter/issues'>" },
+            "at2": { "content": "</a>" },
+            "at3": { "content": "<a target='_blank' href='mailto:admin@dimden.dev'>" }
+        }
+    },
+    "initialization": { "message": "初始化" },
+    "converting": { "message": "转换" },
+    "time_mode": { "message": "在19:00-09:00 之间启用深色模式" },
+    "show_original_images": { "message": "始终显示原始质量的图像（在速度较慢的计算机上可能会滞后）" },
+    "pitch_black_mode": { "message": "漆黑模式（仅当启用深色模式时）" },
+    "hide_retweets": { "message": "隐藏转推" },
+    "see_tweet_views": { "message": "查看推文浏览次数" },
+    "history": { "message": "历史记录" },
+    "autotranslate_tweets": { "message": "自动翻译推文" },
+    "dont_autotranslate": { "message": "不自动翻译推文" },
+    "round_avatars": { "message": "圆形头像" },
+    "search_user_tweets": { "message": "搜索用户推文" },
+    "no_tl_tweets": { "message": "您没有关注任何人，因此没有找到任何推文！ 在 <a href='https://twitter.com/old/settings'>OldTwitter 设置</a>中将时间线类型更改为“算法”。" },
+    "replying_to": { "message": "正在回复", "description": "Used for mention list. Use 'Mentions' if translation sounds bad in your language.", "example": "Replying to: - user1, - user2" },
+    "display": { "message": "显示", "description": "Used for display settings (fonts, colors, style, etc)" },
+    "twitter_blue_checkmarks": { "message": "显示 Twitter Blue 复选标记" },
+    "retweet_hotkey_warn": { "message": "你刚刚按下了 T 键，这是转发的热键。 你想转发这条推文吗？ 如果你按下确认键，你将不会再次看到此警告。" },
+    "reply_btn": { "message": "回复", "description": "Used for reply button." },
+    "retweet_btn": { "message": "转发/引用推文", "description": "Used for retweet menu button." },
+    "like_btn": { "message": "喜欢/不喜欢", "description": "Used for like button." },
+    "favorite_btn": { "message": "收藏/取消收藏", "description": "Used for favorite button." },
+    "views_count": { "message": "浏览量", "description": "Used for views count." },
+    "developer_mode": { "message": "开发者模式" },
+    "copy_user_id": { "message": "复制用户 ID" },
+    "copy_tweet_id": { "message": "复制推文 ID" },
+    "unfollowers": { "message": "取消关注者" },
+    "unfollowings": { "message": "取消关注" },
+    "no_unfollowers": { "message": "你还没有取消任何关注者！" },
+    "no_unfollowings": { "message": "你还没有取消关注任何人！" },
+    "recent_unfollow_update": { "message": "取消关注更新时间不到 15 分钟。" },
+    "deleted_accounts": { "message": "似乎此列表中的每个人都删除了他们的帐户。" },
+    "open_tweet_newtwitter": { "message": "在新 Twitter 中打开推文" },
+    "black_mode_link": { "message": "黑色模式链接" },

```

```suggestion
    "black_mode_link": { "message": "漆黑模式链接" },

```
不知道这个是深色模式（Dark Mode）还是漆黑模式（Black Mode）的配套文案，总之最好还是统一一下